hudi-utilities/src/test/java/org/apache/hudi/utilities/functional/TestHoodieDeltaStreamer.java

```diff
@@ -1115,6 +1125,47 @@ public void testAsyncClusteringServiceWithCompaction() throws Exception {
     });
   }
 
+  @Test
+  public void testHoodieAsyncClusteringJobWithScheduleAndExecute() throws Exception {
```

Comment:
I see a lot of common code between this test and `testHoodieAsyncClusteringJob`. Shall we extract it to common private method? The try-catch part is only different, which can remain in the test method.

Reply:
nice idea, changed